samples/sampler_wrap.html

```diff
@@ -0,0 +1,241 @@
+
+        uniform sampler2D diffuse;
+
+        in vec2 uv;
+
+        layout(location = 0) out vec4 color;
+
+        void main()
+        {
+            color = texture(diffuse, uv);
+        }
+    </script>
+
+    <script>
+
+        (function() {
+            'use strict';
+
+            var canvas = document.createElement('canvas');
+            canvas.width = Math.min(window.innerWidth, window.innerHeight);
+            canvas.height = canvas.width;
+            document.body.appendChild(canvas);
+
+            var gl = canvas.getContext( 'webgl2', { antialias: false } );
+            if (!gl) {
+                gl = canvas.getContext( 'experimental-webgl2', { antialias: false } );
+                document.getElementById('info').innerHTML = 'WebGL 2 is not available.  See <a href="https://www.khronos.org/webgl/wiki/Getting_a_WebGL_Implementation">How to get a WebGL 2 implementation</a>';
+                return;
+            }
+
+            // -- Divide viewport
+
+            var windowSize = {
+                x: canvas.width,
+                y: canvas.height
+            };
+
+            var corners = {
+                TL: 0,
+                TR: 1,
+                BL: 2,
+                BR: 3,
+                MAX: 4
+            };
+
+            var Viewport = new Array(corners.MAX);
+
+            Viewport[corners.TL] = {
+                x: 0,
+                y: 0,
+                z: windowSize.x / 2,
+                w: windowSize.y / 2
+            };
+
+            Viewport[corners.TR] = {
+                x: windowSize.x / 2,
+                y: 0,
+                z: windowSize.x / 2,
+                w: windowSize.y / 2
+            };
+
+            Viewport[corners.BR] = {
+                x: windowSize.x / 2,
+                y: windowSize.y / 2,
+                z: windowSize.x / 2,
+                w: windowSize.y / 2
+            };
+
+            Viewport[corners.BL] = {
+                x: 0,
+                y: windowSize.y / 2,
+                z: windowSize.x / 2,
+                w: windowSize.y / 2
+            };
+
+            // -- Initialize program
+
+            var program = createProgram(gl, getShaderSource('vs'), getShaderSource('fs'));
+
+            var positionLocation = gl.getAttribLocation(program, 'Position');
+            gl.enableVertexAttribArray(positionLocation);
+            var texcoordLocation = gl.getAttribLocation(program, 'Texcoord');
+            gl.enableVertexAttribArray(texcoordLocation);
+
+            var uniformMvpLocation = gl.getUniformLocation(program,"MVP");
+            var uniformDiffuseLocation = gl.getUniformLocation(program,"diffuse");
+
+            // -- Initialize buffer
+            var positions = new Float32Array([
+                -1.0, -1.0,  1.0,
+                -1.0,  1.0,  1.0,
+                 1.0,  1.0, -1.0,
+                 1.0, -1.0, -1.0
+            ]);
+            var vertexBuffer = gl.createBuffer();
```

Comment:
More precisely, this is a `positionBuffer`, not just a `vertexBuffer`.  The other samples call it `vertexPosBuffer`, which is OK too.  Just make them consistent.